### ISSUE TITLE
[Sourcing-2][Accessibility: ACC-264.1][WIP] Table updated not conveyed.

### ISSUE THREAD

**Issue by divyamalhotra488** [SAP/fundamental-ngx#12492]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
Component having issue: https://sap.github.io/fundamental-ngx/#/platform/table/basic
If Table gets updated by any action then the same info in not conveyed to users by screen reader.

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
All latest versions.

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Use the following stackblitz example, and follow:
1. You will see a 'Change Values' button at the bottom of the table in the stackblitz example.
2. Once you click it, it will update table values in UI
3. But the screen reader is not able to announce this change.

#### Please provide stackblitz example(s).
[stackblitz](https://stackblitz.com/edit/prywkj-ovhv1j?file=package.json)

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations.
Yes we have got this bug from Accessibility expert only.

#### Did you check the documentation and the API?
Yes

#### Did you search for similar issues?
Yes

#### Is there anything else we should know?
Reopening previous closed defect (https://github.com/SAP/fundamental-ngx/issues/11257) since the issue is again reported in recent accessibility scan.

**_IMPORTANT: Please refrain from providing links or screenshots of SAP's internal information, as this project is open-source, and its contents are accessible to anyone._**

**Comment by droshev** [comment]
@divyamalhotra488 Can you help me understand what is different than the [issue](https://github.com/SAP/fundamental-ngx/issues/11257)? Especially with [this](https://github.com/SAP/fundamental-ngx/issues/11257#issuecomment-2278214893)

**Comment by divyamalhotra488** [comment]
Hi @droshev, I have created the issue again since https://github.com/SAP/fundamental-ngx/issues/11257 was closed by you and we again got same defect in recent accessibility scan.
I will ask an accessibility team expert to comment here.

**Comment by droshev** [comment]
Similar to #12493 For us (as documented [here](https://github.com/SAP/fundamental-ngx/issues/11257#issuecomment-2278214893)) there is no issue. If you need us to do something you would need to provide different example where the is reproduced.

**Comment by mikerodonnell89** [comment]
The new information is read correctly when the user navigates focus to the updated information. I think in this case, the responsibility to force a screenreader announcement may be on the application developer.

**Comment by divyamalhotra488** [comment]
Hi @mikerodonnell89, the application is using platform table component which is very much encapsulated and cannot be modified from application side.
If it is working as per expectations, I will ask accessibility team to have a look here.

**Comment by MeghanaVernekar** [comment]
Hello Colleagues,

Please find the attached screenshot below of the screen reader announcement for table updated:

![image](https://github.com/user-attachments/assets/e181f35f-7808-4a24-9be2-a72f4896f334)

Please let me know if you need any more details.

Regards,
Meghana.

**Comment by mikerodonnell89** [comment]
> Hi @mikerodonnell89, the application is using platform table component which is very much encapsulated and cannot be modified from application side. If it is working as per expectations, I will ask accessibility team to have a look here.

So obviously in the stackblitz example

> Hello Colleagues,
> 
> Please find the attached screenshot below of the screen reader announcement for table updated:
> 
> ![image](https://private-user-images.githubusercontent.com/172006721/377435516-e181f35f-7808-4a24-9be2-a72f4896f334.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzM0MjE2MTMsIm5iZiI6MTczMzQyMTMxMywicGF0aCI6Ii8xNzIwMDY3MjEvMzc3NDM1NTE2LWUxODFmMzVmLTc4MDgtNGEyNC05YmUyLWE3MmY0ODk2ZjMzNC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjA1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIwNVQxNzU1MTNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wZDNmNzk0YWY2NTQyYzQ1Y2EzYzBmNGQyNjQ1NDQxMTYzZTE2YzYxNGZjMDA1YjNlNjUzZWY2N2MwY2RiZDNiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.NqFMqqZIno0HCG67GaLnvpFY5v-lhrx5twyIQVJ0mek)
> 
> Please let me know if you need any more details.
> 
> Regards, Meghana.

What components are being used here? Is it table + smart filter bar? There are a number of events emitted by the smart filter bar that you can hook in to and make your own custom announcements, or programmatically move focus to the table after a filter event so the screenreader starts reading the data. Search "event" on the API page here to get an idea of what is possible https://fundamental-ngx.netlify.app/#/platform/smart-filter-bar/api

**Comment by mikerodonnell89** [comment]
Closing due to inactivity
